k-distribution/tests/regression-new/imp++-llvm/imp.md

```diff
@@ -0,0 +1,677 @@
+---
+copyright: Copyright (c) 2014-2020 K Team. All Rights Reserved.
+---
+
+IMP++
+=====
+
+Author: Grigore Roșu (grosu@illinois.edu)  
+Organization: University of Illinois at Urbana-Champaign
+
+### Abstract
+This is the **K** semantic definition of the IMP++ language.
+IMP++ extends the IMP language with the features listed below. We
+strongly recommend you to first familiarize yourself with the IMP
```

Comment:
What are the differences between the tutorial version of this and this one?

It seems to me the only difference is `type="Set"` on the `<thread>` cell. Can we fix this in the tutorial instead to just use the LLVM backend? Maybe by adding a unique-thread-id cell to the `<thread>` sub-configuration, and having every new thread spawned used `!X:Int` as that id?

I know you asked about this on Slack, but I didn't realize it was such a small change between the tutorial and this version.

Reply:
It's not the definitions that are the problem. The changes to the definitions are manageable. The problem is the changes to the Makefiles